core/audits/long-tasks.js

```diff
@@ -14,6 +14,9 @@ import {getJavaScriptURLs, getAttributableURLForTask} from '../lib/tracehouse/ta
 
 /** We don't always have timing data for short tasks, if we're missing timing data. Treat it as though it were 0ms. */
 const DEFAULT_TIMING = {startTime: 0, endTime: 0, duration: 0};
+const DISPLAYED_TASK_COUNT = 20;
+// 99% of the May 2023 HTTP Archive run had fewer than 150 long tasks.
+const DEBUG_TASK_COUNT = 150;
```

Comment:
I went back and forth on this number. 150 is _a lot_, but even in the top 1000 most popular sites in CrUX, there are still 18 sites with more than 150 long tasks in their May 2023 HTTP Archive run.

LHR size-wise, the good news is that 90% of the 27 million pages tested in the May run have ~50 long tasks (the media is 11 long tasks). Most pages won't have enough to move the needle.

150 long tasks will cover all long tasks of 99% of pages. Of the remaining 1% of pages, 90% of them only have long tasks after that of less than 75ms (there's only 1 site in the top 1000 with more than 150 long tasks and a 151st long task longer than 75ms).

At that point, we're capturing basically everything that's going on, and I don't think it's worth boosting this number to e.g. 240 to be able to capture the 99.9% case.